addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -253,3 +253,18 @@ $-o-bg-colors-custom: null;
     background-color: $input-focus-bg;
     border-color: $o-input-hover-border-color;
 }
+
+
+// Inputs fields
```

Comment:
I'd call this section "Switches"
Aso, I'd clarify that `o_switch_toggle` should be used in conjunction with the default BS class